demo/vl-radio-group.html

```diff
@@ -0,0 +1,36 @@
+<html>
+
+<head>
+  <title>vl-radio voorbeeld</title>
+  <script type="module" src="demo.js"></script>
+  <script type="module" src="/src/vl-radio.js"></script>
+  <script type="module" src="/src/vl-radio-group.js"></script>
+
+  <link rel="stylesheet" type="text/css" href="/node_modules/vl-ui-core/dist/style.css" />
+</head>
+
+<body>
+  <h1>vl-ui-radio-group demo</h1>
+  <p>
+    <a href="https://overheid.vlaanderen.be/webuniversum/v3/documentation/forms/vl-ui-radio">
+      Meer voorbeelden en documentatie</a>.
+  </p>
+
+  <div>
+    <h2>Radio</h2>
+    <pre>
+      <code class="line-numbers language-markup">
+        &lt;vl-radio-group&gt;
+          &lt;vl-radio id="radio-1" data-vl-label="Ja" data-vl-value="yes"&gt;&lt;/vl-radio&gt;
+          &lt;vl-radio id="radio-2" data-vl-label="Nee" data-vl-value="no"&gt;&lt;/vl-radio&gt;
+        &lt;/vl-radio-group&gt;
+      </code>
+    </pre>
+    <div class="demo">
+      <vl-radio-group>
+        <vl-radio id="radio-1" data-vl-label="Ja" data-vl-value="yes"></vl-radio>
+        <vl-radio id="radio-2" data-vl-label="Nee" data-vl-value="no"></vl-radio>
+      </vl-radio-group>
```

Comment:
misschien in de demo ook tonen dat ge evt nog eerst uw labels zou kunnen buiten de radios zetten? dus dat vl-radio niet de enige children moeten zijn van vl-radio-group. of is dat wel het geval?

Reply:
en stel, puur hypothetisch dat er nog iemand ne vl-radio binnen de vl-radio-group zet en daar ook als name 'name' kiest? of ben ik aan t overdrijven?

Reply:
> misschien in de demo ook tonen dat ge evt nog eerst uw labels zou kunnen buiten de radios zetten? dus dat vl-radio niet de enige children moeten zijn van vl-radio-group. of is dat wel het geval?

Ik begrijp niet zo goed wat je bedoelt of wilt bereiken.

Reply:
> en stel, puur hypothetisch dat er nog iemand ne vl-radio binnen de vl-radio-group zet en daar ook als name 'name' kiest? of ben ik aan t overdrijven?

Codegewijs zal het attribuut tijdens constructie altijd overschreven worden. Hiervoor is geen test voorzien. Als jij dat nodig vindt, kan ik dat toevoegen. Is niet veel werk. Graag nog een seintje dan.

Reply:
<vl-radio-group>
       <label>ik zet hier graag ook nog ne label want da vind ik veel plezanter dan het label mee te geven via vl-radio</label>
        <vl-radio id="radio-1" data-vl-value="yes"></vl-radio>
<br/><!--ik zet hier ook graag nog nen break tussen want da vind ik ook plezant-->
        <vl-radio id="radio-2" data-vl-value="no"></vl-radio>
      </vl-radio-group>

of is dat volledig niet de bedoeling?

Reply:
> > en stel, puur hypothetisch dat er nog iemand ne vl-radio binnen de vl-radio-group zet en daar ook als name 'name' kiest? of ben ik aan t overdrijven?
> 
> Codegewijs zal het attribuut tijdens constructie altijd overschreven worden. Hiervoor is geen test voorzien. Als jij dat nodig vindt, kan ik dat toevoegen. Is niet veel werk. Graag nog een seintje dan.

bwa nee ik zal wel loco aan t gaan zijn

Reply:
Binnen de vl-radio-group wordt alleen gezocht naar vl-radio elementen om het name attribuut te zetten. Het staat de afnemer vrij om daar nog in te renderen wat hij wil. Ik heb hiervoor ook geen test voorzien, maar lijkt me ook niet nodig. Anders hoor ik het nog wel.